src/main/java/domain/Lotteries.java

```diff
@@ -0,0 +1,43 @@
+package domain;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Lotteries {
+
+	private static final int INITIAL_RANKS = 0;
+
+	private final List<Lottery> lotteries;
+
+	public Lotteries(final List<Lottery> lotteries) {
+		this.lotteries = lotteries;
+	}
+
+	public List<Lottery> getLotteries() {
+		return lotteries;
+	}
+
+	public Map<Rank, Integer> getTheNumberOfWinners(final WinningLottery winningLottery) {
+		Map<Rank,Integer> winners = initRankResult();
+		lotteries.forEach(lottery -> {
+			Rank rank = winningLottery.getRank(lottery);
+			putCountToWinner(winners, rank);
+		});
+		return winners;
+	}
+
+	private Map<Rank, Integer> initRankResult() {
+		final Map<Rank, Integer> rankResult = new HashMap<>();
+		Rank.getValues()
+			.forEach((rank) -> rankResult.put(rank, INITIAL_RANKS));
+		return rankResult;
```

Comment:
```suggestion
		return Rank.getValues()
		               .stream()
		               .collect(Collectors.toMap(Function.identity(), value -> INITIAL_RANKS));
```
로 할 수도 있을거 같아요.
그리고 getValues의 경우 values()와 혼동될거 같은데 좀더 자세하게 표현하는게 좋지 않을까요??

Reply:
values와 비슷한 기능을 하기 때문에 getValues로 지었는데 혼동의 여지가 있네요.. 바꾸겠습니다.

Reply:
2차 수정 완료